From 5cd936e48f1378f7a50723ae538bbfda1b53bcb2 Mon Sep 17 00:00:00 2001
From: Hans de Goede <hdegoede@redhat.com>
Date: Fri, 21 Sep 2012 18:57:27 -0300
Subject: [RHEL6 qemu-kvm PATCH 06/23] ehci: remove unused attach_poll_counter

RH-Author: Hans de Goede <hdegoede@redhat.com>
Message-id: <1348253864-3050-6-git-send-email-hdegoede@redhat.com>
Patchwork-id: 42181
O-Subject: [RHEL-6.4 qemu-kvm PATCH 05/22] ehci: remove unused attach_poll_counter
Bugzilla: 805172
RH-Acked-by: Uri Lublin <uril@redhat.com>
RH-Acked-by: Arnon Gilboa <agilboa@redhat.com>
RH-Acked-by: Gerd Hoffmann <kraxel@redhat.com>

From: Gerd Hoffmann <kraxel@redhat.com>

Signed-off-by: Gerd Hoffmann <kraxel@redhat.com>
Upstream commit: 863b474163085a3df13e2c6bf8e696486f3a0c66
Conflicts: hw/usb-ehci.c
---
 hw/usb-ehci.c | 2 --
 1 file changed, 2 deletions(-)

Signed-off-by: Eduardo Habkost <ehabkost@redhat.com>
---
 hw/usb-ehci.c | 2 --
 1 file changed, 2 deletions(-)

diff --git a/hw/usb-ehci.c b/hw/usb-ehci.c
index dac899f..ccf304d 100644
--- a/hw/usb-ehci.c
+++ b/hw/usb-ehci.c
@@ -401,7 +401,6 @@ struct EHCIState {
      */
     QEMUTimer *frame_timer;
     QEMUBH *async_bh;
-    int attach_poll_counter;
     uint32_t astate;                        // Current state in asynchronous schedule
     uint32_t pstate;                        // Current state in periodic schedule
     USBPort ports[NB_PORTS];
@@ -885,7 +884,6 @@ static void ehci_reset(void *opaque)
 
     s->astate = EST_INACTIVE;
     s->pstate = EST_INACTIVE;
-    s->attach_poll_counter = 0;
 
     for(i = 0; i < NB_PORTS; i++) {
         if (s->companion_ports[i]) {
-- 
1.7.11.4

